.github/workflows/ci.yml

```
@@ -0,0 +1,158 @@
+name: CI
+
+on: [push]
```

Although It won't make any difference still it's a good practice to write single events as string & not an array.
```suggestion
on: push
```